src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Are we sure that we don't need to add to clrconfigvalues.h post this change?  Note that clrconfigvalues.h allows while adding a config value whether it is internal/external or a config setting can come from env/registry/config file.  I don't see jitconfigvalues.h has similar sophistication and where do we specify such options if we are adding just in jitconfigvalues.h?

Further when i looked into JIT-EE call getStringConfigValue() and getIntConfigValue() it seems to be calling utilcode to read config value and which would in turn depends on src/inc/clrconfigvalues.h